theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/BasicKeyframedTrack/KeyframeEditor/SingleKeyframeDot.tsx

```diff
@@ -30,34 +30,57 @@ import {
 } from '@theatre/studio/panels/SequenceEditorPanel/DopeSheet/Right/KeyframeSnapTarget'
 
 export const DOT_SIZE_PX = 6
-const DOT_HOVER_SIZE_PX = DOT_SIZE_PX + 5
+const DOT_HOVER_SIZE_PX = DOT_SIZE_PX + 2
 
 const dotTheme = {
   normalColor: '#40AAA4',
   selectedColor: '#F2C95C',
+  inlineEditorOpenColor: '#FCF3DC',
+  selectedAndInlineEditorOpenColor: '#CBEBEA',
 }
 
+const selectBacgroundForDiamond = ({
+  isSelected,
+  isInlineEditorPopoverOpen,
+}: IDiamond) => {
+  if (isSelected && isInlineEditorPopoverOpen) {
+    return dotTheme.inlineEditorOpenColor
+  } else if (isSelected) {
+    return dotTheme.selectedColor
+  } else if (isInlineEditorPopoverOpen) {
+    return dotTheme.selectedAndInlineEditorOpenColor
+  } else {
+    return dotTheme.normalColor
+  }
+}
+
+type IDiamond = {isSelected: boolean; isInlineEditorPopoverOpen: boolean}
```

Comment:
Thought: In some other places, we've been considering calling this kind of type the `ThemeValues` type alias, and just passing the entire `const themeValues: ThemeValues = { ... }` into the `styled.div<{theme: ThemeValues}>`. Then, you can just create the theme values once and so forth for this and the below `<HitZone theme={themeValues}>`.

Non-blocking. If this sounds good to you, try it out. If it needs further explanation feel free to wave in Tandem!

Reply:
@colelawrence It sounds like a good idea, but I don't feel like we reached the complexity that would require us to introduce the `ThemeValues` type alias yet. Also, if it affects other places, then I'd handle this refactor in another PR. What do you think about it? Would you be comfortable with us not using the type alias here? :relaxed: 

I really don't  have much against it, I just don't think we need it (yet). If you still think that the lack of the type alias is a non-blocking issue, then I'd merge this PR, otherwise I can make the required changes tomorrow! 😁